pp/integrations/plugin-packs/procedures/operatingsystems-freebsd-snmp.md

```diff
@@ -314,154 +312,154 @@ yum install centreon-plugin-Operatingsystems-Freebsd-Snmp
+| WARNING      | Warning threshold                                                                                                                                                                               | 80                                                      |             |
+| CRITICAL     | Critical threshold                                                                                                                                                                              | 90                                                      |             |
+| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options).                                                                                              | --verbose --filter-perfdata='storage.space\|used\|free' |             |
 
 </TabItem>
 <TabItem value="Disk-IO" label="Disk-IO">
 
-| Macro         | Description                                                                                                                            | Default value     | Mandatory   |
-|:--------------|:---------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
-| DISKNAME      | Set the storage name, it can be a regex (empty means 'check all storage')                                                              |                   |             |
-| CRITICALREAD  | Critical threshold                                                                                                                     |                   |             |
-| WARNINGREAD   | Warning threshold                                                                                                                      |                   |             |
-| CRITICALWRITE | Critical threshold                                                                                                                     |                   |             |
-| WARNINGWRITE  | Warning threshold                                                                                                                      |                   |             |
+| Macro         | Description                                                                                        | Default value     | Mandatory   |
+|:--------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| DISKNAME      | Set the device (number expected) example: 1, 2,... (empty means 'check all devices')               |                   |             |
```

Comment:
```suggestion
| DISKNAME      | Set the storage name, it can be a regex (empty means 'check all storage')             |                   |             |
```